metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py

```diff
@@ -418,9 +418,17 @@ def loop_tables(
                 self.report.report_dropped(dataset_name)
                 continue
 
-            columns = inspector.get_columns(table, schema)
-            if len(columns) == 0:
-                self.report.report_warning(dataset_name, "missing column information")
+            try:
```

Comment:
@shirshanka should I remove this or it can stay here. 
It make sure for sources to handle gracefully if there is a problem parsing a column. This does not belong strictly to this pr, so I think maybe this should be in a separate pr if you are ok with it.
This helps to not restart the whole source from the beginning if it turns out there is an invalid column information.

Reply:
lgtm